bpaotu/bpaotu/static/bpaotu/bpaotu.js

```diff
@@ -499,31 +499,41 @@ $(document).ready(function() {
     };
 
     function check_for_auth_token() {
-        if (authentication_token == null) {
+        if (window.otu_search_config['ckan_auth_integration'] == "False") {
```

Comment:
minor: shouldn't this be coming through as a boolean, rather than doing a string comparison?

Reply:
Fixed.